README.md

```diff
@@ -96,7 +97,22 @@ Connect to the container (with KUBECONFIG exported)
 make connect
 ```
 
-Accessing OKD UI
+To access the OKD UI from the host running `docker`, remember to export `OKD_CONSOLE_PORT=443` before `make cluster-up`.
+You should find out the IP address of the OKD docker container
 ```
-TODO - in the process of working out the details here.
+clusterip=$(docker inspect $(docker ps |grep quay.io/kubevirtci/ocp-4.3|awk '{print $1}')  | jq '.[0].NetworkSettings.IPAddress')
+```
+and make it known in `/etc/hosts` via
+```
+cat << EOF >> /etc/hosts 
+$cluster_ip console-openshift-console.apps.test-1.tt.testing
+$cluster_ip oauth-openshift.apps.test-1.tt.testing
+EOF
+```
+Now you can browse to https://console-openshift-console.apps.test-1.tt.testing 
+and log in by picking the `htpasswd_provider` option. The credentials are `admin/admin`.
+
+```
+TODO add tunneling tricks to allow browsing from remote hosts
+iptables -t nat -A PREROUTING -p tcp -i em3 --dport 443 -j DNAT --to-destination $cluster_ip
```

Comment:
port name worth parmetrize or writing a comment on change it if the if name is different ?

Reply:
I dropped the example as I did not test, and left only the TODO.